plugins/ephemeral_tabs.lua

```diff
@@ -6,17 +6,35 @@ local DocView = require "core.docview"
 local Doc = require "core.doc"
 local TreeView = require "plugins.treeview"
 
+local function is_called_by(module)
+  local i = 1
+  while debug.getinfo(i) do
+    if debug.getinfo(i).source:match(module .. "%.lua$") then
```

Comment:
If you're dead set on using the debug function, at least avoid double calling and use the selector:
```lua
debug.getinfo(i, "S")
```
This will at least increase performance.

Reply:
I still recommend to find another way, as using `debug.getinfo` is not a great idea.

Reply:
> If you're dead set on using the debug function

Suggest another solution.

Reply:
I haven't tested it, but what was wrong with the solution I proposed before?

Reply:
Do you mean setting a global flag on a mouse event? In my opinion, this is a crutch.
We have a call stack. Consider that these are ready-made global flags for each function. What is the problem with using it?

Reply:
Are you sure using a _local_ variable is worse that bothering the debug library?

Reply:
I'm not sure if "worse" is the right word here. Imagine if something like match.round was implemented in lua, and you offered me to use your own implementation for a small range of numbers. Well it looks like this to me.

Reply:
It feels more like over-complicating a simple solution...